src/partition/test-repart.sh

```diff
@@ -8,6 +8,15 @@ set -o pipefail
 repart="${1:?}"
 test -x "$repart"
 
+sfdisk=
+for i in /usr/bin /usr/sbin /bin /sbin; do
```

Comment:
maybe just append /usr/sbin to PATH?

Reply:
Yeah, maybe just do
`command -v sfdisk >/dev/null || PATH=$PATH:/sbin:/usr/sbin` ?

Reply:
I'd just do it unconditionally, i don't think there's any purpose in not having sbin in the path these days